README.md

```diff
@@ -22,6 +22,8 @@ The available configuration options are detailed below.
 |:--------|:-----------|
 | StatusHost | IP address or DNS of the NGINX+ instance to retrieve status information from. Defaults to `localhost`. |
 | StatusPort | Port the NGINX+ status endpoint can be reached at. Defaults to `8080`. |
+| LegacyAPI | Indicate whether the NGINX+ is using `legacy` API or not. |
```

Comment:
```suggestion
| LegacyAPI | Indicate whether the NGINX+ is using `legacy` API or not. Defaults to `false`. |
```

Reply:
Hello @rmfitzpatrick,

As per the latest design(that exists in the current source code), we have removed the config option of  `LegacyAPI`.
We are currently driving the logic of determining `legacy` or `newer` APIs based on the single config option of `APIVersion`. A value of zero(0) in the `APIVersion` represents `legacy` APIs and a positive integer value in the `APIVersion` represents the API version of `newer` APIs.

Please let us know in case of any queries